components/drata/actions/create-asset/create-asset.mjs

```diff
@@ -0,0 +1,105 @@
+import drata from "../../drata.app.mjs";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/AssetsPublicController_createAsset/";
+
+export default {
+  key: "drata-create-asset",
+  name: "Create Asset",
+  description: `Create an asset. [See docs here.](${docsLink})`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    drata,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the asset",
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "The description of the asset",
+    },
+    assetClassTypes: {
+      type: "string[]",
+      label: "Asset Class Types",
+      description: "The asset class types",
+      options: [
+        "HARDWARE",
+        "POLICY",
+        "DOCUMENT",
+        "PERSONNEL",
+        "SOFTWARE",
+        "CODE",
+        "CONTAINER",
+        "COMPUTE",
+        "NETWORKING",
+        "DATABASE",
+        "STORAGE",
+      ],
+    },
+    assetType: {
+      type: "string",
+      label: "Asset Type",
+      description: "The asset type",
+      options: [
+        "PHYSICAL",
+        "VIRTUAL",
+      ],
+    },
+    ownerId: {
+      type: "integer",
```

Comment:
Does Drata provide an API for listing owners?

Reply:
I didn't find one. Also user, owner, personnel, and vendors `id`s are all a mysterious gray area to me

Reply:
I applied `personnelId` to this one